.cicd/platforms/ubuntu20.Dockerfile

```diff
@@ -6,11 +6,13 @@ RUN apt-get update && apt-get upgrade -y && \
                        cmake                \
                        git                  \
                        jq                   \
-                       libboost-all-dev     \
                        libcurl4-openssl-dev \
                        libgmp-dev           \
                        libssl-dev           \
                        llvm-11-dev          \
                        ninja-build          \
+                       python3-all          \
                        python3-numpy        \
+                       ubuntu-dev-tools     \
+                       zlib1g-dev           \
```

Comment:
Remember to update README with this addition

Reply:
done